__tests__/utils/locale.js

```diff
@@ -75,7 +75,7 @@ describe('Locale Utils', () => {
       'en_EN',
```

Comment:
This file should not have changes.

Reply:
> This file should not have changes.



I didn't change it, you changed it to create a search test, you told me to rebase it, it was added, I didn't change it

Reply:
But this PR should **not contain this**. It's already in master. This is stuff you commited.